src/lib/hooks/routing/useClientSideSmartOrderRouterTrade.ts

```diff
@@ -10,7 +11,16 @@ import { getClientSideQuote } from './clientSideSmartOrderRouter'
 import { useRoutingAPIArguments } from './useRoutingAPIArguments'
 
 const protocols: Protocol[] = [Protocol.V2, Protocol.V3]
-const config = { protocols }
+
+// The following configs are used to reduce latency when using the client side SOR
```

Comment:
Instead of making this a const, in-line it in the config object, and just add a comment to every line of the config explaining why you're setting it as you are and what it does, so it's easier to reason through in the future.